grunt/config/verify-zip-size.js

```diff
@@ -14,34 +42,73 @@ module.exports = function( grunt ) {
 		"Checks the size of the created artifact and creates an issue if the zip exceeds 5MB",
 		async function() {
 			const done = this.async();
+
 			const stats = fs.statSync( "artifact.zip" );
-			if ( stats.size >= 5242880 ) {
-				const data = {
-					title: "Fix RC zip size",
-					body: "The RC zip size should be smaller than 5MB.",
-					labels: [
-						"type:development",
-						"component:tools",
-					],
-				};
-
-				// TODO: get the github token from somewhere.
-				const url = `https://api.github.com/repos/Yoast/wordpress-seo/issues?access_token=${ GITHUB_TOKEN }`;
-				// TODO: Fix the curl command: probably some encoding that needs to be done?
-				execSync( `curl ${ url } --data ${ JSON.stringify( data ) } --header "application/vnd.github.symmetra-preview+json"` );
-
-				// Send a message to the slack plugin channel.
-				const slackWebhook = new IncomingWebhook( process.env.SLACK_DEV_PLUGIN_CHANNEL_TOKEN );
-
-				const sizeInMB     = ( stats.size / 1024 / 1024 ).toFixed( 2 );
-				await slackWebhook.send( {
-					text: `Zip size is too big, it is ${ sizeInMB } mb. [LINK TO ISSUE]`,
-				} );
 
+			if ( stats.size < 5242880 ) {
 				done();
+			}
+
+			const data = {
+				title: "Fix RC zip size",
+				body: "The RC zip size should be smaller than 5MB.",
+				labels: [
+					"type:development",
+					"component:tools",
+					"fix-before-release",
+				],
+			};
 
-				grunt.fail.fatal( `Zip size is too big (${ stats.size } bytes). The release process is stopped.` );
+			const versionString = grunt.option( "plugin-version" );
+			const version = parseVersion( versionString );
+
+			const milestoneTitle = ( version.patch > 0 ) ? `hotfix/${ versionString }` : `release/${ versionString }`;
+			const milestone = await getMilestone( milestoneTitle );
+
+			if ( milestone ) {
+				data.milestone = milestone.number;
+			}
+
+			const response = await fetch( `${ API_BASE }/issues`, {
```

Comment:
Maybe we can extract this to a function: githubRequest or something like that. We can just co: githubRequest(  data ) and the function will handle authorization and that kind of stuff. It's something we should consider. 

Also, by having that function we are probably able to create the release branch by the api instead of relying on github commands :)